src/test/mochitest/browser_dbg-breakpoints.js

```diff
@@ -73,3 +73,37 @@ add_task(async function() {
   bp2 = findBreakpoint(dbg, "simple2", 5);
   is(bp2.disabled, false, "second breakpoint is enabled");
 });
+
+add_task(async function() {
+  Services.prefs.setBoolPref("devtools.features.skip-pausing", true);
+
+  const dbg = await initDebugger("doc-scripts.html");
+
+  await selectSource(dbg, "simple2");
+  await addBreakpoint(dbg, "simple2", 3);
+  await addBreakpoint(dbg, "simple2", 5);
+
+  let syncedBps = waitForDispatch(dbg, "SYNC_BREAKPOINT", 2);
+  await reload(dbg, "simple2");
+  await waitForSelectedSource(dbg, "simple2");
+  await syncedBps;
+
+  assertNotPaused(dbg);
+});
+
+add_task(async function() {
+  Services.prefs.setBoolPref("devtools.features.skip-pausing", false);
+
+  const dbg = await initDebugger("doc-scripts.html");
+
+  await selectSource(dbg, "simple2");
+  await addBreakpoint(dbg, "simple2", 3);
+  await addBreakpoint(dbg, "simple2", 5);
+
+  let syncedBps = waitForDispatch(dbg, "SYNC_BREAKPOINT", 2);
+  await reload(dbg, "simple2");
+  await waitForSelectedSource(dbg, "simple2");
+  await syncedBps;
+
+  assertPaused(dbg);
+});
```

Comment:
we try to avoid multiple `add_task` calls in the same test file because they run at the same time in different tabs. It might be nicer to have a new mochitest file and  consolidate the two tasks.

Reply:
Thanks for the tip!